### ISSUE TITLE
Cannot create post with a tags that have "space" character

### ISSUE THREAD

**Issue by rmdwirizki** [adamhaile/surplus-realworld#1]
I think you should change the trigger event to input a tag with "enter" key instead of "space"

**Comment by adamhaile** [comment]
You're right, I hadn't noticed that the demo worked that way.  I think the React one split on space?  Or maybe I just had that wrong :).  I'll try and get a fix together soon.
